crates/prelude/src/clock.rs

```diff
@@ -101,6 +101,14 @@ impl<H: Handler> Timer<H> {
         core::mem::forget(self);
     }
 
+    /// Returns true if the timer has elapsed.
+    pub fn is_elapsed(&self) -> bool {
```

Comment:
This function is not needed. It can be implemented by the user.

Reply:
Removed this.